config/puma.rb

```diff
@@ -19,16 +27,12 @@
 threads_count = ENV.fetch('RAILS_MAX_THREADS', 3)
 threads threads_count, threads_count
 
-# Specifies the `worker_timeout` threshold that Puma will use to wait before
-# terminating a worker in development environments.
-#
-worker_timeout 3600 if ENV.fetch('RAILS_ENV', 'development') == 'development'
-
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 port ENV.fetch('PORT', 3000)
-pidfile ENV.fetch('PIDFILE') { 'tmp/pids/server.pid' }
-state_path ENV.fetch('STATEPATH') { 'tmp/pids/server.state' }
-bind ENV.fetch('SOCKETFILE') { 'unix://tmp/sockets/puma.sock' }
 
-# Allow puma to be restarted by `rails restart` command.
+# Allow puma to be restarted by `bin/rails restart` command.
 plugin :tmp_restart
+
+# Specify the PID file. Defaults to tmp/pids/server.pid in development.
+# In other environments, only set the PID file if requested.
+pidfile ENV['PIDFILE'] if ENV['PIDFILE']
```

Comment:
We lost the config for state_path and socketfile config here, but I think we want to keep these?

Reply:
Good catch, fixed.